masterlist.yaml

```
@@ -2784,31 +2784,102 @@ plugins:
+        util: 'FO4Edit v4.0.4'
+  - name: 'EvenBetterMDesc.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/36105/'
+        name: 'Even Better Mod Descriptions'
+    msg:
+      - <<: *alsoUseX
+        subs: [ '[Even Better Mod Descriptions Fixed](https://www.nexusmods.com/fallout4/mods/74459/)' ]
+        condition: 'checksum("EvenBetterMDesc.esp", 31185F01)'
     clean:
       # Original
-      - crc: 0x3BC4243F
+      - crc: 0x31185F01
         util: 'FO4Edit v4.0.4'
-      # Merged with UFO4P patch as suggested on the mod page
-      - crc: 0xDC398CCF
+      # Fixed
+      - crc: 0x31BE007A
+        util: 'FO4Edit v4.0.4'
+  - name: 'EvenBMD_AutomatronDLC.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/36105/'
+        name: 'Even Better Mod Descriptions'
+    msg:
+      - <<: *alsoUseX
+        subs: [ '[Even Better Mod Descriptions Fixed](https://www.nexusmods.com/fallout4/mods/74459/)' ]
+        condition: 'checksum("EvenBMD_AutomatronDLC.esp", 51338E69)'
+    clean:
+      # Original
+      - crc: 0x51338E69
         util: 'FO4Edit v4.0.4'
       # Fixed
-      - crc: 0xC0BEAF59
+      - crc: 0x384AF106
+        util: 'FO4Edit v4.0.4'
+  - name: 'EvenBMD_FarHarborDLC.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/36105/'
+        name: 'Even Better Mod Descriptions'
+    msg:
+      - <<: *alsoUseX
+        subs: [ '[Even Better Mod Descriptions Fixed](https://www.nexusmods.com/fallout4/mods/74459/)' ]
+        condition: 'checksum("EvenBMD_FarHarborDLC.esp", 38F38215)'
+    clean:
+      # Original
+      - crc: 0x38F38215
+        util: 'FO4Edit v4.0.4'
+      # Fixed
+      - crc: 0xA1D256BB
+        util: 'FO4Edit v4.0.4'
+  - name: 'EvenBMD_NukaWorldDLC.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/36105/'
+        name: 'Even Better Mod Descriptions'
+    msg:
+      - <<: *alsoUseX
+        subs: [ '[Even Better Mod Descriptions Fixed](https://www.nexusmods.com/fallout4/mods/74459/)' ]
+        condition: 'checksum("EvenBMD_NukaWorldDLC.esp", 38F38215)'
+    clean:
+      # Original
+      - crc: 0x48971FBB
+        util: 'FO4Edit v4.0.4'
+      # Fixed
+      - crc: 0xBEB20B31
         util: 'FO4Edit v4.0.4'
-    # Can skip adding other as this is the main variant and has most downloads.
   - name: 'EBMD-UF04P Patch.esp'
     url:
       - link: 'https://www.nexusmods.com/fallout4/mods/36105/'
         name: 'Even Better Mod Descriptions'
     msg:
       - <<: *alreadyInOrFixedByX
         subs: [ 'Even Better Mod Descriptions GOTY Fixed](https://www.nexusmods.com/fallout4/mods/74459/)' ]
-        condition: 'checksum("EBMD_AllInOne.esp", C0BEAF59)'
+        condition: 'checksum("EBMD_AllInOne.esp", F0DDE373) or checksum("EBMD_AllInOne.esp", 08B0A737)'
     clean:
-      - crc: 0x504E4A32
+      # Original
+      - crc: 0x50AC05D1
+        util: 'FO4Edit v4.0.4'
+      # Fixed
+      - crc: 0xCFD5B0B2
         util: 'FO4Edit v4.0.4'
```

Keep newline between mods.
```suggestion
        util: 'FO4Edit v4.0.4'

```